views/NewChart.react.js

```diff
@@ -5,10 +5,17 @@ import ChartForm from './forms/ChartForm'
 
 import { createChart } from './fe-utils/fetchUtil'
 
+import { routes } from './routes/routes'
+
 const NewChart = () => {
   const handleSubmit = async (e, formValues) => {
-    e.preventDefault()
-    await createChart(formValues)
+    try {
+      e.preventDefault()
+      const { data } = await createChart(formValues)
+      window.location.replace(routes.chart(data.chart_id))
```

Comment:
should we use `assign` instead of `replace`? If we use `replace` then the user can't go back in the browser.

Reply:
[7fbf5cb](https://github.com/AMP-SCZ/dpdash/pull/88/commits/7fbf5cbc1947e95c1407fac03868022cb1bd158b)